metadata-ingestion/tests/integration/sql_server/golden_files/golden_mces_mssql_no_db_to_file.json

```diff
@@ -47,8 +55,75 @@
     "changeType": "UPSERT",
     "aspectName": "subTypes",
     "aspect": {
-        "value": "{\"typeNames\": [\"Database\"]}",
-        "contentType": "application/json"
+        "json": {
+            "typeNames": [
+                "Database"
+            ]
+        }
+    },
+    "systemMetadata": {
+        "lastObserved": 1615443388097,
+        "runId": "mssql-test"
+    }
+},
+{
+    "entityType": "dataFlow",
+    "entityUrn": "urn:li:dataFlow:(mssql,localhost.Weekly Demo Data Backup,PROD)",
+    "changeType": "UPSERT",
+    "aspectName": "dataFlowInfo",
+    "aspect": {
+        "json": {
+            "customProperties": {},
+            "externalUrl": "",
+            "name": "localhost.Weekly Demo Data Backup"
+        }
+    },
+    "systemMetadata": {
+        "lastObserved": 1615443388097,
+        "runId": "mssql-test"
+    }
+},
+{
+    "entityType": "dataJob",
+    "entityUrn": "urn:li:dataJob:(urn:li:dataFlow:(mssql,localhost.Weekly Demo Data Backup,PROD),localhost.Weekly Demo Data Backup)",
+    "changeType": "UPSERT",
+    "aspectName": "dataJobInfo",
+    "aspect": {
+        "json": {
+            "customProperties": {
+                "job_id": "d29457a6-e8df-4a69-8f55-37414e74317c",
+                "job_name": "Weekly Demo Data Backup",
+                "description": "No description available.",
+                "date_created": "2023-03-01 14:46:32.447000",
```

Comment:
Since "job_id", "date_created" and "date_modified" will change for every run of the test, you will have to pass `ignore_paths` inside `tests/integration/sql_server/test_sql_server.py` when calling `check_golden_file`

Example:
```

mce_helpers.check_golden_file(
                ... other inputs....
                ignore_paths=[
                    r"root\[\d+\]\['aspect'\]\['json'\]\['customProperties'\]\['job_id'\]",
                    r"root\[\d+\]\['aspect'\]\['json'\]\['customProperties'\]\['date_created'\]",
                    r"root\[\d+\]\['aspect'\]\['json'\]\['customProperties'\]\['date_modified'\]",
                ],
            )
```